plugin/mishPlugin/mishPlugin.h

```diff
@@ -0,0 +1,59 @@
+#ifndef _MISH_PLUGIN_H
+#define _MISH_PLUGIN_H
+
+#include "NvInfer.h"
+#include "NvInferPlugin.h"
+
+namespace nvinfer1
+{
+	class MishPlugin : public IPluginExt
```

Comment:
We are only taking plugin contributions based on IPluginV2Ext interfaces (IPluginV2DynamicExt)

Reply:
HI, @rajeevsrao  I want to ask about the mish activation function  in YOLO v4 that either its fully implemented and available in newer versions of tensorRT or not ... like tensorRT 6 or 7.  because after conversion from FP32 to FP16 format, the accuracy degradation is observed which means some layers are not fully implemented which results in performance drop. Thanks